src/normalize/text.ts

```diff
@@ -40,25 +40,21 @@ export default function normalizeText(editor: Editor, entry: NodeEntry): boolean
     }
 
     if (Title.isTitle(node)) {
+        // Title shouldn't be followed by another title.
+        const [next, nextPath] = Editor.next(editor, { at: path }) ?? []
+        if (Title.isTitle(next)) {
+            Transforms.mergeNodes(editor, { at: nextPath })
+            return true
+        }
+
         // Title must be the first child of it's parent.
-        // If it is found in the figure, move it.
-        // In other cases, change it to paragraph.
         const [parent] = Editor.parent(editor, path)
         const isFirst = path[path.length - 1] === 0
-        if (Figure.isFigure(parent)) {
-            // Figure must not have more than one title.
-            const [next, nextPath] = Editor.next(editor, { at: path }) ?? []
-            if (Title.isTitle(next)) {
-                Transforms.mergeNodes(editor, { at: nextPath })
-                return true
-            }
-
-            if (!isFirst) {
-                const newPath = [...path]
-                newPath[newPath.length - 1] = 0
-                Transforms.moveNodes(editor, { at: path, to: newPath })
-                return true
-            }
+        if (Figure.isFigure(parent) && !isFirst) {
+            const newPath = [...path]
+            newPath[newPath.length - 1] = 0
+            Transforms.moveNodes(editor, { at: path, to: newPath })
+            return true
```

Comment:
This is already handled in `normalizeFigure`.

Reply:
without this:
```
1) normalization
       figure
         title-wrong-order:
     TypeError: Cannot read property 'includes' of undefined
      at match (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/transforms/node.ts:732:37)
      at Object.from [as nodes] (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/interfaces/editor.ts:674:12)
      at nodes.next (<anonymous>)
      at Function.from (<anonymous>)
      at fn (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/transforms/node.ts:162:30)
      at Object.withoutNormalizing (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/interfaces/editor.ts:1318:5)
      at Object.liftNodes (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/transforms/node.ts:147:12)
      at _loop (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/transforms/node.ts:730:20)
      at fn (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/transforms/node.ts:721:39)
      at Object.withoutNormalizing (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/interfaces/editor.ts:1318:5)
      at Object.unwrapNodes (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/transforms/node.ts:699:12)
      at normalizeInvalidChild (src/normalize/figure.ts:129:16)
      at Object.normalizeOrderedChildren (src/normalize/util.ts:55:13)
      at Object.normalizeFigure [as default] (src/normalize/figure.ts:42:13)
      at normalizeNode (src/normalize/index.ts:27:27)
      at fn (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/interfaces/editor.ts:762:16)
      at Object.withoutNormalizing (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/interfaces/editor.ts:1318:5)
      at Object.normalize (.yarn/cache/slate-https-75f3649d72-577f9a6e41.zip/node_modules/slate/src/interfaces/editor.ts:749:12)
      at fn (test/index.js:13:12)
      at Context.runTest (test/util/fixtures.js:32:17)
      at processImmediate (internal/timers.js:461:21)

```